### DISCUSSION TITLE
Output the estimated carbon emission of IQ-TREE

### DISCUSSION THREAD

**Discussion by trongnhanuit** [iqtree/iqtree2#341] [Ideas]
Would it be useful to output the estimated carbon emission at the end of each IQ-TREE run?This would demonstrate our awareness of environmental impacts and our efforts to reduce the carbon footprint from our work (by speeding up IQ-TREE).

**Comment by roblanf** [comment]
If it's simple enough to estimate, I don't see any reason _not_ to do this!
